src/app/containers/Header/index.jsx

```diff
@@ -1,15 +1,46 @@
 import React, { useContext } from 'react';
 import SkipLink from '@bbc/psammead-brand/skip-link';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import BrandContainer from '../Brand';
 import NavigationContainer from '../Navigation';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
+import { UserContext } from '#contexts/UserContext';
+import { getOtherVariant } from '#lib/utilities/variantHandler';
 import ConsentBanner from '../ConsentBanner';
 import useToggle from '../Toggle/useToggle';
 
+const renderScriptLink = (
+  script,
+  service,
+  scriptLink,
+  setPreferredVariantCookie,
+  variant,
+) => {
+  const { text, offscreenText } = scriptLink;
+  const otherVariant = getOtherVariant(service, variant);
+
+  return (
+    <ScriptLink
+      script={script}
+      service={service}
+      href={`/${service}/${otherVariant}`}
+      variant={otherVariant}
+      onClick={() => setPreferredVariantCookie(service, otherVariant)}
+    >
+      <span aria-hidden="true">{text}</span>
+      <VisuallyHiddenText> {offscreenText} </VisuallyHiddenText>
```

Comment:
` ${offscreenText} `

to add the extra spacing in the text ^ and do we need the space on the right